app/src/test/java/decide/LIC10Test.java

```diff
@@ -0,0 +1,103 @@
+package decide;
+
+import decide.datastructures.Parameters;
+import decide.datastructures.Point;
+import decide.lic.LIC10;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class LIC10Test {
+
+    // Negative test
+    @Test public void lic10FalseSetOfLessThan5Points () {
+        LIC10 LIC = new LIC10();
+        Parameters parameters = new Parameters();
+        Point[] points = {
+                new Point(0,0),
+                new Point(1,0),
+                new Point(5,3),
+                new Point(-2,4)
+        };
+        // Parameters: AREA1, E_PTS, F_PTS
+        parameters.AREA1 = 1;
+        parameters.E_PTS = 1;
+        parameters.F_PTS = 1;
+        assertFalse("False test case with set of 4 points", LIC.compute(points, parameters));
+    }
+
+    // Negative test
+    @Test public void lic10FalseLargeGaps () {
```

Comment:
I am not sure if we have to test for invalid input, as the instructions say that `E_PTS+F_PTS ≤ NUMPOINTS−3`. But it depends on how we should interpret the instructions.

If we test for it, we should add one more point to this test, as the length is currently 4 instead of 5. The compute function will return on points.length < 5 instead of the for loop where it checks the gaps with `i < points.length - parameters.E_PTS - parameters.F_PTS - 2`

Otherwise LGTM